### PR TITLE
feat: Added mongoose schema

### PR DESCRIPTION
**Pull request by thesanjeevsharma**
Review the schema, so I can proceed on this with GQL types and resolvers!

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const path = require("path");
+const cors = require("cors");
+const graphqlHTTP = require('express-graphql');
+const mongoose = require('mongoose');
+const app = express();
+
+// MongoDB Connection
+mongoose.connect('mongodb://localhost:27017/recipe-app', { useUnifiedTopology : true, useNewUrlParser : true })
```
Comment (prvnbist) [inline server/app.js]:
```suggestion
mongoose.connect(process.env.DB_URI', { useUnifiedTopology : true, useNewUrlParser : true })
```

File: server/models/ingredient.model.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+const IngredientSchema = new mongoose.Schema({
+    valid: {
```
Comment (prvnbist) [inline server/models/ingredient.model.js]: Use this convention whenever there's a boolean
```suggestion
    isValid: {
```

File: server/models/sachet.model.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        default : false
+    },
+    tracking : {
+        type : Boolean,
+        required : ['Tracking value is required.']
+    },
+    modes : [{
+        type : {
+            type : String,
+            enum : ['Real Time', 'Co-Packer', 'Planned Lot'],
```
Comment (prvnbist) [inline server/models/sachet.model.js]: Add a default value for this field

File: server/app.js (hunk 1 above)
Comment (thesanjeevsharma) [reply server/app.js]: Oh, it's temporary!

**Pull request merged**